revisions/N0/src/text/dialog_4.asm

```diff
@@ -401,8 +401,7 @@ Dialog1B9::
 
 Dialog1BA::
     db "I DISSE TRAKTER "
-    db "ER HEMMELIGHETER"
-    db "NÆRE@"
+    db "DØLGES DET@"
```

Comment:
Litt for obskurt? Føler det passer uglestatuene litt, og er arkaisk på litt samme måte som NIGH

Reply:
Liker din variant

Reply:
fisefint med DØLGES ja